### PR TITLE
Fixing links which opens in new tab (where don't required)

### LINKED ISSUE #458: Don't open links in new tab.
Please open links in the same tab, e.g. GCI pages.

### PR DESCRIPTION
**Pull request by subhahu123**
# Changes done in this Pull Request

Removed the target attribute which is opening the links in new tabs

- Fixes #458 

### Before:

peek 2018-11-25 12-37

### CODE REVIEW

**Comment by Ryan10145** [comment]
@subhahu123 There are still target attributes in the social media icons, do you think those should be removed as well? There are still a lot of things that open up on different pages, such as all of the social media icons and any links that are in the widgets.

**Comment by Ryan10145** [comment]
Also please provide a preview link

**Comment by subhahu123** [comment]
@Ryan10145  i thing those should be open in other tabs
that's why but u want i will do that also

File: _includes/header.html [hunk 1]
```diff
@@ -85,16 +85,16 @@
               <li><a href="/#mentors">Mentors</a></li>
               <li><a href="/#students">Students</a></li>
               <li ><a href="/#contributors">Contributors</a></li>
-              <li><a href="students_projects.html" target="_blank">Projects</a></li>
+              <li><a href="students_projects.html" >Projects</a></li>
               <li><a href="http://gci18.fossasia.org/#connect">Connect</a></li>
               <li><a href="#map">Map</a></li>
               <li class="code-in-nav has-dropdown"><a href="https://codein.withgoogle.com/organizations/fossasia/"
-                  target="_blank">GCI</a>
```
Comment (jogendra) [inline _includes/header.html]: Keep it open in new tab

File: _includes/header-pages.html [hunk 2]
```diff
@@ -80,13 +80,13 @@
           <div class="module left">
             <ul class="menu">
               <li class="code-in-nav has-dropdown"><a href="https://codein.withgoogle.com/organizations/fossasia/"
-                  target="_blank">GCI</a>
+                  >GCI</a>
```
Comment (jogendra) [inline _includes/header-pages.html]: Keep this open in new tab, it is external URL

**Comment by abishekvashok** [comment]
Can you please fix the rest as well, or should I mrge this PR and not close the issue.

**Comment by subhahu123** [comment]
> Can you please fix the rest as well, or should I mrge this PR and not close the issue.

i tried to resolve conflict but it is showing no conflicts in gti and gitkoren
i have tried both cli and gui

**Pull request closed without merging**